langs/wren/stdin.patch

```diff
@@ -1,7 +1,7 @@
---- wren/src/cli/vm.c
-+++ wren/src/cli/vm.c
+--- src/cli/vm.c
++++ src/cli/vm.c
 @@ -303,11 +303,35 @@ static void freeVM()
- 
+
```

Comment:
I think the whitespace is intentional in a patch? Maybe just leave it, I expected this just to be a file rename.